eng/pipelines/official.yml

```diff
@@ -1,13 +1,11 @@
 pr: none
 trigger:
-  batch: false
-  branches:
+ branches:
     include:
-      - dev
-      - release-*-MSRC
+      - main
```

Comment:
I don't think you should be blindly following their recommendation. 
We use dev, not main.

Reply:
Sure, pushed changes. Please check now.